.travis.yml

```diff
@@ -66,13 +66,15 @@ before_install:
   - bash "${CI_SCRIPTS}/run-epics-iocs-on-procserv.sh"
 
 install:
+  - |
+    if [[ $TRAVIS_PYTHON_VERSION == 'nightly' ]]; then
+      # Nightly has some issues with h5py
+      sed -i -e '/h5py/d' test-requirements.txt
+    fi
```

Comment:
This doesn't fill me with fuzzy feelings

Reply:
@tacaswell will installing `h5py-dev` help here or is there something else?